test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,34 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsContainsNull() {
+        Object nullObj = null;
+        // some object contains null
+        assertContainsNull("", 1, new ArrayList<>(), nullObj, "cc");
+        assertContainsNull("", 1, new ArrayList<>(), nullObj);
+        assertContainsNull(nullObj);
+        assertContainsNull("a", nullObj);
+
+        // all objects must not contain null
```

Comment:
`must not` sounds a bit weird :) Let's try `// all objects without null` or `// all objects don't contain null`